src/HttpServer.h

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include "ThreadPool.h"
+
+using namespace std;
+
+public class HttpServer
+{
```

Comment:
Is this class copyable? If not, please delete its copy constructor and assignment operator.